fv3core/stencils/d_sw.py

```diff
@@ -63,6 +63,36 @@ def flux_adjust(w: sd, delp: sd, gx: sd, gy: sd, rarea: sd):
         w = flux_integral(w, delp, gx, gy, rarea)
 
 
+@gtstencil()
+def horizontal_relative_vorticity_from_winds(
+    u: FloatField,
+    v: FloatField,
+    ut: FloatField,
+    vt: FloatField,
+    dx: FloatField,
+    dy: FloatField,
+    rarea: FloatField,
+    vorticity: FloatField,
+):
+    """
+    Compute the area mean relative vorticity in the z-direction from the D-grid winds.
+
+    Args:
+        u (in): x-direction wind on D grid
+        v (in): y-direction wind on D grid
+        ut (out): u * dx
+        vt (out): v * dy
```

Comment:
we can see immediately below that ut and vt are vector windspeed * del (cell). it doesn't hurt, but without words to describe these, do we need this?

Reply:
This is where I'd look to figure out what things are, and writing this docstring I did have to take time to read every line of code to see the input data is entirely overwritten and only overwritten with that value. I think even with code below saying what these are, it's better having this than having nothing.

In the longer term when more stencils are joined, these are intermediate values which should probably exist within a single gtscript function, instead of being passed between two functions/stencils.